jest.config.js

```diff
@@ -11,10 +11,11 @@ module.exports = {
       statements: 43,
     },
   },
-  setupFiles: ['./test/setup.js', './test/env.js'],
-  setupFilesAfterEnv: ['./test/jest/setup.js'],
-  testMatch: [
-    '<rootDir>/ui/**/?(*.)+(test).js',
-    '<rootDir>/shared/**/?(*.)+(test).js',
-  ],
+  // TODO: enable resetMocks
+  // resetMocks: true,
+  restoreMocks: true,
+  setupFiles: ['<rootDir>/test/setup.js', '<rootDir>/test/env.js'],
+  setupFilesAfterEnv: ['<rootDir>/test/jest/setup.js'],
+  testMatch: ['<rootDir>/ui/**/*.test.js', '<rootDir>/shared/**/*.test.js'],
+  testTimeout: 2500,
```

Comment:
This looks new - is this the default setting?

Reply:
We bumped the timeout at one point in the module template: https://github.com/MetaMask/metamask-module-template/blob/main/jest.config.js#L26